### PR TITLE
test: add some tests for code scanning

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/integrate/provider/sonar/sonar_test.go [hunk 1]
```diff
@@ -25,3 +27,47 @@ func Test_extractCeTaskId(t *testing.T) {
 
 	}
 }
+
+func Test_findJavaBinaryFiles(t *testing.T) {
```
Comment (supereagle) [inline pkg/integrate/provider/sonar/sonar_test.go]: Camel case?

File: pkg/util/os/os_test.go [hunk 2]
```diff
@@ -63,3 +64,62 @@ func TestMustGet(t *testing.T) {
 	assert.Equal(t, f, 11.1)
 
 }
+
+func Test_IsFileExists(t *testing.T) {
```
Comment (supereagle) [inline pkg/util/os/os_test.go]: Same here

**Comment by zhujian7** [comment]
@supereagle  All comments have been fixed. PTAL

**Pull request merged**